pkg/daemon/ceph/client/pg.go

```diff
@@ -31,17 +35,31 @@ type PGDumpBrief struct {
 	ActingPrimaryID int    `json:"acting_primary"`
 }
 
-func GetPGDumpBrief(context *clusterd.Context, clusterName string) ([]PGDumpBrief, error) {
+func GetPGDumpBrief(context *clusterd.Context, clusterName string, isNautilus bool) (*PGDumpBrief, error) {
+	var pgDump PGDumpBrief
+	var pgStats PgStats
 	args := []string{"pg", "dump", "pgs_brief"}
 	buf, err := NewCephCommand(context, clusterName, args).Run()
 	if err != nil {
 		return nil, fmt.Errorf("failed to get pg dump: %+v", err)
 	}
 
-	var pgDump []PGDumpBrief
+	//this is to check the content of buf, which is empty in case of mimic for nautilus it works fine
+	logger.Infof("buf %v", string(buf))
```

Comment:
buf is empty in case of mimic

Reply:
hmm, i would have expected the difference to only be during the Unmarshal call.